### PR TITLE
Fix stackstorm Dockerfile

### PR DESCRIPTION
**Pull request by mccricardo**
stackstorm Dockerfile needed some changes in order to build correctly.

### CODE REVIEW

File: stackstorm/Dockerfile [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 LABEL com.stackstorm.version="${ST2_VERSION}"
 
-ADD ./sources.list /etc/apt/ 
+ADD ./sources.list /etc/apt/
 
-RUN apt-get -y update && apt-get -y install adduser procps python2.7 libffi5 libyaml-0-2 git libpcre3 libpython2.7 libxml2 python-dev libssl-dev libffi-dev sudo
+RUN apt-get -y update && apt-get -y install wget adduser procps python2.7 libffi5 libyaml-0-2 git libpcre3 libpython2.7 libxml2 python-dev libssl-dev libffi-dev sudo
 
-ADD ${ST2_PACKAGE} /tmp/st2_${ST2_VERSION}_amd64.deb
+RUN wget ${ST2_PACKAGE} -O /tmp/st2_${ST2_VERSION}_amd64.deb
```
Comment (armab) [inline stackstorm/Dockerfile]: `ADD` works fine too for remote URLs: https://docs.docker.com/engine/reference/builder/#add

File: stackstorm/Dockerfile [hunk 2]
```diff
@@ -1,16 +1,16 @@
 FROM debian:wheezy
 
-ARG ST2_VERSION
-ARG ST2_REPO=stable
+ARG ST2_VERSION=1.6.0-2
```
Comment (armab) [inline stackstorm/Dockerfile]: These things are parametrized, to avoid editing it for every new release

**Comment by armab** [comment]
You can do stuff with parameters: 
```sh
docker build --build-arg ST2_VERSION="1.6.0-2" ST2_REPO="staging-stable" -t st2 stackstorm/
```
Example from the `CI`: https://github.com/StackStorm/st2-dockerfiles/blob/master/.circle/docker.sh#L49

Apart of that, there are `1.6dev` images ready: https://hub.docker.com/r/stackstorm/st2api/tags/
Do you have issues with them?

**Comment by armab** [comment]
So @mccricardo Thanks for digging in!

This worked for me: 

```sh
git clone https://github.com/StackStorm/st2-dockerfiles.git
docker build --build-arg ST2_VERSION="1.6.0-3" --build-arg ST2_REPO="staging-stable" -t st2 stackstorm/

CONTAINERS="st2actionrunner st2api st2auth st2stream st2exporter st2notifier st2resultstracker st2rulesengine st2sensorcontainer st2garbagecollector"

for container in $CONTAINERS; do
  docker build -t stackstorm/${container}:latest ${container}
done
```

What's your use case, what you're trying to build? Did https://hub.docker.com/r/stackstorm/ work for you?

**Comment by armab** [comment]
Besides of that, I created an Issue: https://github.com/StackStorm/st2-dockerfiles/issues/20 for the `RED` build.

I can say we officially support Docker, but instructions in previous message ^^ can help you to do the things.

File: stackstorm/Dockerfile (hunk 2 above)
Comment (mccricardo) [inline stackstorm/Dockerfile]: I see. I just though a sensible default might make sense. But I understand your reasoning.

File: stackstorm/Dockerfile (hunk 1 above)
Comment (mccricardo) [inline stackstorm/Dockerfile]: Yes it does! I changed this but when testing I tested multiple stuff at once. Like you pointed out, not needed.

**Comment by mccricardo** [comment]
Excellent! Thank you for your feedback. I'll be closing this PR.

`docker-compose up -d` fails because of issue  #16, although I managed to got it working by building locally the missing containers. At the end some containers had exited and were not working but I haven't, yet, had the chance get into it. I'll be digging in shortly. 

Once again, thank you!

**Pull request closed without merging**